pandas/io/tests/test_clipboard.py

```diff
@@ -71,6 +71,7 @@ def check_round_trip_frame(self, data_type, excel=None, sep=None):
     def test_round_trip_frame_sep(self):
```

Comment:
True, but if the implementation changes, we would like to make sure such behaviour does not break.  In any case, couldn't hurt, right? :smile: